Kernel/System/CustomerUser/LDAP.pm

```diff
@@ -1067,10 +1067,22 @@ sub CustomerSearchDetail {
     );
 
     if ( $ResultSearch->code() ) {
-        $Kernel::OM->Get('Kernel::System::Log')->Log(
-            Priority => 'error',
-            Message  => $ResultSearch->error(),
-        );
+        if ( $ResultSearch->code() == 4 ) {
```

Comment:
Hi @pboguslawski,

thanks for your PR. 
I have checked the CPAN module. [LINK](https://metacpan.org/pod/Net::LDAP::Constant#SYNOPSIS)

Maybe we can use the variable `LDAP_SIZELIMIT_EXCEEDED` instead of the number 4. 

```
use Net::LDAP qw(LDAP_SIZELIMIT_EXCEEDED);
```

Can you please change that?
Maybe you can find a way to check this in a unit test.

Thanks 🚀

Reply:
> Can you please change that?

Done.

> Maybe you can find a way to check this in a unit test.

Unit tests was not changed; feel free to extend it if you see the point in doing it in this case.